Automattic/pocket-casts-android
# Display Ratings story

## Description

This adds ratings story UI.

Figma: lH66LwxxgG8btQ8NrM0ldx-fi-3070_21275

## Testing Instructions

1. Sign in with an account that listened to at least 5 minutes of episodes this year.
2. Sync your profile data.
3. Close the app.
4. Reopen the app.
5. Open Playback 2024.
6. You might see a loading progress bar and it might take some time to sync your data.
7. Once data is synced go to the ratings story and verify it.

## Screenshots or Screencast 

### Preview

| High ratings | Low ratings | No ratings | No ratings small device |
| - | - | - | - |
| ![high](https://github.com/user-attachments/assets/55c88835-edf3-4f99-94a2-86d27ce78508) | ![low](https://github.com/user-attachments/assets/642ae455-35e1-4055-b36f-6719eb462adf) | ![Screenshot_20241016-172428](https://github.com/user-attachments/assets/51840efa-c9c3-400a-923b-f8d1c23cef30) | ![Screenshot 2024-10-16 at 17 25 31](https://github.com/user-attachments/assets/8a6c603d-4e10-40f5-b2e8-a2fed033be52) |

### Ratings animation

https://github.com/user-attachments/assets/38b86b62-05b2-4532-bec8-ca33dc7e816c

### No ratings animation

https://github.com/user-attachments/assets/9c756c8f-fd70-4e17-9b11-3159d4e92c55

## Checklist
 
#### I have tested any UI changes...
